cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -35,9 +35,8 @@ func (suite *InstallerIntegrationTestSuite) TestInstallFromLocalSource() {
 		e2e.AppendEnv(constants.DisableUpdates+"=false"))
 
 	// Assert output
-	cp.Expect("Installing State Tool")
+	cp.Expect("Installing Update")
```

Comment:
This is wrong; the test is testing a fresh install, not a update. The fact that this is failing suggests either fresh installs are giving the wrong UX or the test is mocked wrong (eg. targeting an install dir that's already being used).